updater/sis-updater-worker/src/updater/updateMeta.js

```diff
@@ -3,14 +3,13 @@ const _ = require('lodash')
 
 const { bulkCreate, selectFromByIdsOrderBy } = require('../db')
 const { Course, CourseProvider, CourseType, CreditType, Organization, StudyrightExtent } = require('../db/models')
+const { universityOrgId } = require('../../../../services/backend/config/organisationConstants')
```

Comment:
Updater and backend run in separate containers so this is not going to work. Even though the different services are stored in a common monorepo, they do not currently share any code between them.

Reply:
Oh yea we need to think other solution for this

Reply:
Added new env variable `ROOT_ORG_ID` to overwrite it. Could this work?